core/execution.go

```diff
@@ -223,7 +224,111 @@ func StaticCall(env vm.Environment, caller vm.ContractRef, addr common.Address,
 
 // Create creates a new contract with the given code
 func Create(env vm.Environment, caller vm.ContractRef, code []byte, gas, gasPrice, value *big.Int) (ret []byte, address common.Address, err error) {
-	// Depth check execution. Fail if we're trying to execute above the limit.
+	nonce := env.Db().GetNonce(caller.Address())
```

Comment:
Should not be changing the logic of Create, the previous changes were done to fix bugs within a simulated environment